plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -1133,29 +1133,32 @@ class Errata(UnitMixin, ContentUnit):
 
     @property
     def rpm_search_dicts(self):
+        metadata = self.create_legacy_metadata_dict()
```

Comment:
What do you think about this suggestion:
Just get `pkglist` from its collection like it's done in [`create_legacy_metadata_dict`](https://github.com/werwty/pulp_rpm/blob/518d0bef0d3e7ad1dce2fcd707b23fe11be1c9df/plugins/pulp_rpm/plugins/db/models.py#L1172). This will let you iterate the cursor and there will be no need to load all the erratum pkglist in one more place and to use some additional legacy :) structure.
Let me know if I miss something here and you think it's better to use `create_legacy_metadata_dict` method